docs/partials/readmeAddlInfo.md

```diff
@@ -0,0 +1,35 @@
+## Turborepo Overview
+
+This monorepo is managed using [Turborepo](https://turborepo.org/).
+
+### Managing dependencies
+
+#### Best practices for dependency installation
+
+When you install a dependency in a component or package in `auro-formkit`, you should install it directly in the package that uses it. 
+
+The package's `package.json` will have every dependency that it needs. This is true for both external and internal dependencies.
+
+- External dependencies come from the `npm` registry.
+
+- Internal dependencies let you share functionality within your repository.
+
+This practice has several benefits:
+
+- **Improved clarity:** It's easier to understand what a package depends on when its dependencies are listed in its `package.json`. Developers working in the repository can see at a glance what dependencies are used within the package.
+
+- **Enhanced flexibility:** In a monorepo at scale, it can be unrealistic to expect each package to use the same version of an external dependency.
+
+- **Better caching ability:** If you install too many dependencies in the root of your repository, you'll be changing the workspace root whenever you add, update, or delete a dependency, leading to unnecessary cache misses.
+
+- **Pruning unused dependencies:** When dependencies are installed in the packages that they are meant for, Turborepo can read your lockfile and remove dependencies that aren't used in the packages you need.
+
+For more information, see the [Turborepo docs](https://turbo.build/repo/docs/crafting-your-repository/managing-dependencies).
```

Comment:
I am not sure if I agree that explaining why we made decisions is best done in the readme. Let's leave it for now but I would likely discuss this section further before we take the component out of beta.